docs/rule/_TEMPLATE_.md

```diff
@@ -1,53 +1 @@
-## (rule name goes here)
-
-(context about the problem goes here)
-
-(what the rule does goes here)
-
-### Examples
-
-This rule **forbids** the following:
-
-```hbs
-{{!-- Example 1  --}}
-```
-
-```hbs
-{{!-- Example 2  --}}
-```
-
-This rule **allows** the following:
-
-```hbs
-{{!-- Example 1  --}}
-```
-
-```hbs
-{{!-- Example 2  --}}
-```
-
-### Migration
-
-(suggest any fast/automated techniques for fixing violations in a large codebase)
-
-* suggestion on how to fix violations using find-and-replace / regexp
-* suggestion on how to fix violations using a codemod
-
-### Configuration
-
-(exclude this section if the rule has no extra configuration)
-
-* object -- containing the following properties:
-  * string -- `parameterName1` -- description of parameter including the possible values and default value
-  * boolean -- `parameterName2` -- description of parameter including the possible values and default value
-
-### Related Rules
-
-* [related-rule-name1](related-rule-name1.md)
-* [related-rule-name2](related-rule-name2.md)
-
-### References
-
-* (link to relevant documentation goes here)
-* (link to relevant function spec goes here)
-* (link to relevant guide goes here)
+Please see dev/new-rule/templates/doc.md to find the documentation template for new rules.
```

Comment:
Do we still need this file?

Reply:
yes, since it's where people expect to find it (via @bmish earlier review)